public/src/smart_resize.js

```diff
@@ -0,0 +1,46 @@
+import { throttle } from 'lodash';
+
+module.exports = function (editor) {
+  let resize = editor.resize;
+  let lastDemensions = [];
+
+  let checkDelay = 250;
+  let recheckDelay = 500;
+  let stableAfter = 5000;
+  let timeouts = [];
+
+  let stableChecks = 0;
+  let maxStableChecks = stableAfter / recheckDelay;
+
+  function exec() {
+    timeouts.splice(0).map(clearTimeout);
```

Comment:
this made my head heart. Really. If we just check every 250ms ,max 5000 we can just have one timer no? then only difference between requestResize and scheduleRecheck will be resetting the counter...